hand-written.md

```diff
@@ -340,15 +365,18 @@ The [SAM conversion](http://www.scala-lang.org/files/archive/spec/2.12/06-expres
     val sam1: MySam = () => 2 // Uses SAM conversion, not the implicit
     sam1.i()                  // Returns 2
 
-To retain the old behavior, you may compile under `-Xsource:2.11`, use an explicit call to the conversion method, or disqualify the type from being a SAM (e.g. by adding a second abstract method).
+To retain the old behavior, your choices are:
+
+* compile under `-Xsource:2.11`
+* use an explicit call to the conversion method
+* disqualify the type from being a SAM (e.g. by adding a second abstract method).
```

Comment:
need to check if this will work with scala-lang's markdown parser.. the rest of the file uses a different style for bullets.